source/engine/ui/editor/impl/EditorCamera.cpp

```diff
@@ -0,0 +1,53 @@
+#include "EditorCamera.h"
```

Comment:
As it is now, the "both" cameras have their transforms tied. Can we cache the transform data of each as we toggle so that moving one camera does not also move the other?